lnclient/ldk/ldk.go

```diff
@@ -1725,6 +1680,22 @@ func (ls *LDKService) deleteOldLDKPayments() {
 	now := time.Now()
 	for _, payment := range payments {
 		paymentCreatedAt := time.Unix(int64(payment.CreatedAt), 0)
+
+		deletablePaymentKind := false
+		switch (payment.Kind).(type) {
+		case ldk_node.PaymentKindBolt11:
+			deletablePaymentKind = true
+		case ldk_node.PaymentKindSpontaneous:
+			deletablePaymentKind = true
```

Comment:
Should we also include `PaymentKindBolt11Jit`? That leaves `PaymentKindOnchain`, `PaymentKindBolt12Offer` and `PaymentKindBolt12Refund`

Reply:
We currently don't support Bolt11Jit (This is LDK's Bolt8 implementation of just-in-time channels https://github.com/lightning/blips/blob/master/blip-0052.md).